.github/workflows/ci-lite-test-full.yml

```diff
@@ -83,9 +83,11 @@ jobs:
         pip list
 
     - name: DocTests Lite
-      if: ${{ matrix.pkg-name != 'LAI' }}
       working-directory: src
-      run: python -m pytest lightning_lite
+      run: |
+        pip install pytest -q
```

Comment:
Same question here

Reply:
Will mark this as resolved as it has the same question/answer, so we can keep just one sample :)

Reply:
Repeating the question can be helpful in case you do one change, it ensures you don't miss out any other occurrences. It's not meant as spam.

Reply:
that is good point, and I ma fine with it, but lets keep conversation just with one